app/test_with_resources.bzl

```diff
@@ -1,20 +1,22 @@
-def test_with_resources(file):
-    """
-    Genrule for test files.
+"""
+Macros for app module resources tests.
```

Comment:
```suggestion
Macros for app module tests that depend on resources.
```

Reply:
done

Reply:
resolving on behalf of ben as it's a nit change.